python/test_2.py

```diff
@@ -27,49 +32,53 @@ def pack_oorbArray(orbits):
     epoch_type = np.zeros(len(orbits)) + 3
     gval = np.zeros(len(orbits)) + 0.15
     # Also, the orbitID has to be a float, rather than a string, so substitute if needed.
-    if ((isinstance(orbits['objid'][0], float) == True) |
-        (isinstance(orbits['objid'][0], int) == True)):
+    if isinstance(orbits['objid'][0], numbers.Real):
         orbids = orbits['objid']
     else:
         orbids = np.arange(0, len(orbits['objid']), 1)
     # Convert to format for pyoorb, INCLUDING converting inclination, node, argperi to RADIANS
     oorbArray = np.column_stack((orbids, orbits['q'], orbits['e'], np.radians(orbits['i']),
-                                    np.radians(orbits['node']), np.radians(orbits['argperi']),
-                                    orbits['t_p'], elem_type, orbits['t_0'], epoch_type, orbits['H'], gval))
+                                 np.radians(orbits['node']), np.radians(orbits['argperi']),
+                                 orbits['t_p'], elem_type, orbits['t_0'], epoch_type, orbits['H'], gval))
     return oorbArray
 
 
 if __name__ == "__main__":
 
     t = time.time()
-    print "starting..."
+    print("starting...")
 
     # check against OpenOrb command line
     timespan = 1000
     timestep = 10
-    command = "/bin/rm test_out ; oorb --code=807 --task=ephemeris --orb-in=test_orbits.des --timespan=%d --step=%d > test_out" %(timespan, timestep)
-    print command
+    try:
+        os.unlink('test_out')
+    except OSError:
+        pass
+    command = "oorb --code=807 --task=ephemeris --orb-in=test_orbits.des " \
+        " --timespan=%d --step=%d > test_out" % (timespan, timestep)
+    print(command)
     subprocess.call(command, shell=True)
```

Comment:
Consider using `subprocess.check_output` with the command as a sequence. Then you don't have to go through the pain of writing the `test_out` file (and therefore do not have to unlink it) and reading it back. You can get the output directly. Is the problem that to parse the output you need to call `read_table` and that can only be given a file name?